frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -87,7 +80,12 @@ export const HeaderContainer = styled.header`
 
 export const ParametersAndCardsContainer = styled.div`
   flex: auto;
+  overflow-y: ${({ shouldMakeDashboardHeaderStickyAfterScrolling }) =>
+    shouldMakeDashboardHeaderStickyAfterScrolling ? "auto" : "visible"};
   overflow-x: hidden;
+  @supports (overflow-x: clip) {
+    overflow-x: clip;
```

Comment:
For the rest [10% of browsers](https://caniuse.com/?search=overflow%3Aclip) that don't support `overflow-x: clip`, we just don't make the parameter sticky at all (please note that in different layouts where we don't hide the dashboard header, the parameters will still be sticky together with the dashboard header). [Bruno is okay with this](https://metaboat.slack.com/archives/C505ZNNH4/p1686054411768389?thread_ts=1686054215.543739&cid=C505ZNNH4).

#### 90% of browsers
<img width="1451" alt="image" src="https://github.com/metabase/metabase/assets/1937582/d4c833db-43f4-453b-8178-a8e969e9401a">

#### 10% of browsers
<img width="1450" alt="image" src="https://github.com/metabase/metabase/assets/1937582/37586b34-4547-44a9-9775-d39326e836fa">